src/features/offer/components/SameArtistPlaylist/api/fetchOffersByArtist.ts

```diff
@@ -0,0 +1,43 @@
+import { captureAlgoliaError } from 'libs/algolia/fetchAlgolia/AlgoliaError'
+import { offerAttributesToRetrieve } from 'libs/algolia/fetchAlgolia/buildAlgoliaParameters/offerAttributesToRetrieve'
+import { client } from 'libs/algolia/fetchAlgolia/clients'
+import { env } from 'libs/environment'
+import { HitOffer, Offer } from 'shared/offer/types'
+
+export type FetchOfferByArtist = {
+  artist: string | null | undefined
```

Comment:
Ca peut être un string composé de plusieurs artistes, aussi je propose que tu nommes ta variable
```suggestion
  artists: string | null | undefined
```

Reply:
C'est d'accord cependant l'API renvoie bien `offer.artist` et non `offer.artists` du coup ce ne serait pas contradictoire d'avoir dans le code artists et artist au niveau de l'API ?

Reply:
Du coup cela va fausser le typage... 
Sinon je peux faire `artist: artists`

Reply:
Si alors il faudrait en discuter avec l'API, mais c'est pas normal comme erreur de typo en effet.
T'es d'accord @lixxday  ?